server/src/test/java/org/elasticsearch/plugins/PluginsServiceTests.java

```diff
@@ -601,7 +601,7 @@ public void testNonExtensibleDep() throws Exception {
     }
 
     public void testIncompatibleElasticsearchVersion() throws Exception {
-        PluginInfo info = new PluginInfo("my_plugin", "desc", "1.0", Version.V_6_0_0,
+        PluginInfo info = new PluginInfo("my_plugin", "desc", "1.0", Version.fromId(6000099),
```

Comment:
why is this better?

Reply:
The test just needs a version that will be considered incompatible with the current version. We could just as well use "1.0.0" since that will never again be compatible with any future version. Here I just converted the old constant use into an arbitrary version.

Reply:
I see. OK. thanks for clarifying